### PR TITLE
Port to .NET 6.0

### PR DESCRIPTION
**Pull request by Megasware128**
The internal GenerateOuterColumn method signature changed including the addition of a table reference expression. I've updated the signature grabbed the table reference and it seems to be working according to your tests.

### CODE REVIEW

**Comment by Rwing** [comment]
Thank you very much, I've been too busy lately, I will merge and release the new package ASAP

File: EntityFrameworkCore.UseRowNumberForPaging/SqlServer2008QueryTranslationPostprocessorFactory.cs [hunk 1]
```diff
@@ -96,10 +102,13 @@ private Expression VisitSelect(SelectExpression selectExpression)
...
 						: new[] { new OrderingExpression(new SqlFragmentExpression("(SELECT 1)"), true) };
-					_ = selectExpression.PushdownIntoSubquery();
+					selectExpression.PushdownIntoSubquery();
 					var subQuery = (SelectExpression)selectExpression.Tables[0];
 					var projection = new RowNumberExpression(Array.Empty<SqlExpression>(), rowOrderings, oldOffset.TypeMapping);
-					var left = GenerateOuterColumnAccessor(subQuery, projection, "row");
+					var generateOuterColumnAccessorDelegate = GenerateOuterColumnAccessorMethodInfo.CreateDelegate(GenerateOuterColumnAccessorDelegateType, subQuery);
+					var tableReferences = (IEnumerable<object>)TableReferencesFieldInfo.GetValue(selectExpression);
+					var tableReferenceExpression = tableReferences.Single();
+					var left = (ColumnExpression)generateOuterColumnAccessorDelegate.DynamicInvoke(tableReferenceExpression, projection, "row", true);
```
Comment (Rwing) [inline EntityFrameworkCore.UseRowNumberForPaging/SqlServer2008QueryTranslationPostprocessorFactory.cs:111]: There may be some performance issues with this method, is it possible to add a cache or  another solution?

Comment (Rwing) [reply EntityFrameworkCore.UseRowNumberForPaging/SqlServer2008QueryTranslationPostprocessorFactory.cs:111]: Sorry, after my testing there doesn't seem to be a performance issue

**Comment by Megasware128** [comment]
Thank you for releasing my changes. The description on the NuGet page is still saying EFCore 5.0. Maybe change that to 6.0 to avoid some confusion?

**Comment by Rwing** [comment]
yep, thank you again.

**Pull request merged**